FirebaseAuth/Sources/Swift/Utilities/AuthErrorUtils.swift

```diff
@@ -551,7 +550,7 @@ class AuthErrorUtils: NSObject {
   #endif // os(iOS)
 
   static func recaptchaSDKNotLinkedError() -> Error {
-    // TODO(chuanr): point the link to GCIP doc once available.
+    // TODO(chuanr, ObjC): point the link to GCIP doc once available.
```

Comment:
nit: Is `ObjC` unique enough to search for these without `TODO`? If not we'll need to remember to search for `TODO(chuanr, ObjC)` and `TODO(ObjC)` to find these. (ignore this comment if it won't impact your workflow)

Reply:
I would do `git grep TODO | grep ObjC`